### PR TITLE
[IT-876] Move cfn-ssm-param-macro Lambda to New Repo

### PR DESCRIPTION
**Pull request by cascadianblue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,88 +1,100 @@
...
-Template for creating lambda repositories
-A template for quickly starting a new AWS lambda project.
+# cfn-ssm-param-macro
 
-## Naming
-Naming conventions:
-* for a vanilla Lambda: `lambda-<context>`
-* for a Cloudformation Transform macro: `cfn-transform-<context>`
-* for a Cloudformation Custom Resource: `cfn-cr-<context>`
+The `SsmParam` macro allows retrieval of parameters from the SSM Parameter Store.
```
Comment (zaro0508) [inline README.md:3]: A cloudformation macro is the same as a transform.  Lets follow the naming convention and rename this to `cfn-transform-ssm-param`

Comment (zaro0508) [reply README.md:3]: i changed my mind, the documentation seems to refer to macros as an instance of transform.  I think it's more appropriate to change the name to `cfn-macro-sssm-param`

Comment (cascadianblue) [reply README.md:3]: `cfn-macro-sssm-param` or `cfn-macro-ssm-param` ? I don't see where the extra "s" would come from

Comment (zaro0508) [reply README.md:3]: that's a typo @cascadianblue should be `cfn-macro-ssm-param`

File: README.md [hunk 2]
```diff
@@ -1,88 +1,100 @@
...
+The Serverless Application Model Command Line Interface (SAM CLI) is an extension of the AWS CLI that adds functionality for building and testing Lambda applications. It uses Docker to run your functions in an Amazon Linux environment that matches Lambda. It can also emulate your application's build environment and API.
 
-```shell script
-$ python -m pytest tests/ -v
+## Use the SAM CLI to build and test locally
+
+Build your application with the `sam build --use-container` command.
+
+```bash
+cfn-macro-ssm-param$ sam build --use-container
```
Comment (zaro0508) [inline README.md:56]: i don't think you need `cfn-macro-ssm-param$` in these bash example

**Pull request merged**